<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="2" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="358" started-at="2016-03-31T13:48:21Z" finished-at="2016-03-31T13:48:22Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="358" started-at="2016-03-31T13:48:21Z" finished-at="2016-03-31T13:48:22Z">
      <class name="p04_mockito.Ex03ThrowingExceptionsFromMocks">
        <test-method status="PASS" signature="shouldThrowExceptionWhenReadingFromSourceThrowsException()[pri:0, instance:p04_mockito.Ex03ThrowingExceptionsFromMocks@161cd475]" name="shouldThrowExceptionWhenReadingFromSourceThrowsException" duration-ms="342" started-at="2016-03-31T15:48:21Z" finished-at="2016-03-31T15:48:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shouldThrowExceptionWhenReadingFromSourceThrowsException -->
        <test-method status="PASS" signature="shouldThrowExceptionWhenSavingToDestinationThrowsException()[pri:0, instance:p04_mockito.Ex03ThrowingExceptionsFromMocks@161cd475]" name="shouldThrowExceptionWhenSavingToDestinationThrowsException" duration-ms="5" started-at="2016-03-31T15:48:22Z" finished-at="2016-03-31T15:48:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shouldThrowExceptionWhenSavingToDestinationThrowsException -->
      </class> <!-- p04_mockito.Ex03ThrowingExceptionsFromMocks -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
